recipes/credentials.rb

```diff
@@ -0,0 +1,26 @@
+#
+# Cookbook Name:: osl-jenkins
+# Recipe:: default
+#
+# Copyright 2014, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+jenkins = encrypted_data_bag_item('ssh-keys', 'osl-jenkins')
```

Comment:
It would be preferred to encase this in a rescue block. This would help provide a useful error in case it can't retrieve the databag or the databag doesn't exist.